src/ring_theory/ideal/local_ring.lean

```diff
@@ -284,6 +283,19 @@ end
 
 end
 
+lemma of_surjective [comm_semiring R] [local_ring R] [comm_semiring S] [nontrivial S]
```

Comment:
I think I would like to preserve the old version that doesn't need `is_local_ring_hom`. Or at least write a comment explaining that you can get rid of that assumption.
But my preference would be to have `of_surjective'` for the commutative ring case.

Reply:
Sure. I retrieved the old version.

Reply:
Can you update the description of the PR?

Reply:
Thank! I have updated.